public_html/wp-content/plugins/camptix-attendance/addons/attendance.php

```diff
@@ -336,6 +397,14 @@ public function setup_controls( $section ) {
 		// Fields
 		$camptix->add_settings_field_helper( 'attendance-enabled', esc_html__( 'Enabled', 'wordcamporg' ), 'field_yesno', 'general' );
 
+		$camptix->add_settings_field_helper(
+			'attendance-qr-enabled',
+			esc_html__( 'QR Code Scanning', 'wordcamporg' ),
+			'field_yesno',
+			'general',
+			esc_html__( 'Enable QR code scanning?', 'wordcamporg' )
+		);
```

Comment:
As much as I prefer this wrapping in multiple lines, maybe write it in one line like the surrounding lines.